pom.xml

```diff
@@ -120,11 +119,6 @@
         <artifactId>bcprov-jdk15on</artifactId>
         <version>${bouncycastle.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.codehaus.plexus</groupId>
-        <artifactId>plexus-utils</artifactId>
-        <version>${plexus-utils.version}</version>
-      </dependency>
       <dependency>
```

Comment:
I would add `maven-shared-utils` as direct dependency (because it is now used directly) and not only rely on it being available transitively through `maven-common-artifact-filters`